### PR TITLE
Tweaks to docs and comments

### PR DESCRIPTION
**Pull request by almarklein**
After being inactive on this for a bit, the fresh look helps to make some improvements.

### CODE REVIEW

File: docs/guide.rst [hunk 1]
```diff
@@ -4,30 +4,30 @@ The jupyter_rfb guide
...
 
-The provided :class:`RemoteFrameBuffer <jupyter_rfb.RemoteFrameBuffer>` class cannot do much by itself, but it serves as
+The provided :class:`RemoteFrameBuffer <jupyter_rfb.RemoteFrameBuffer>` class cannot do much by itself, but it provides
 a basis for widgets that want to generate images at the server, and be informed
 of user events. The way to use ``jupyter_rfb`` is therefore to create a subclass
-and implement two methods.
+and implement two specific methods.
 
-The first method to implement is ``get_frame()``, which should return an (uint8) numpy array:
+The first method to implement is ``.get_frame()``, which should return an (uint8) numpy array. For example:
```
Comment (djhoese) [inline docs/guide.rst:30]:
```suggestion
The first method to implement is ``.get_frame()``, which should return a uint8 numpy array. For example:
```

I'm not sure the parentheses do anything useful here and I could be wrong about the `an` to `a` but I don't think so (if you say "you-int" then it is a Y sound which is not a vowel sound so it should be `a`).

File: docs/conf.py [hunk 2]
```diff
@@ -42,7 +42,14 @@ def insert_examples():
         f.write("\n".join(lines).encode())
 
 
+def other_doc_tweaks():
+    """Tweak some docstrings to make them more suited for the html docs."""
+    m = jupyter_rfb.RemoteFrameBuffer.handle_event
+    m.__doc__ = m.__doc__.replace("the docs of ``jupyter_rfb.events``", "below")
```
Comment (djhoese) [inline docs/conf.py:48]: I've never seen someone use this hack before. Why not link to the document or whatever it is directly with something like:

```rst
the docs of :mod:`jupyter_rfb.events`.
```

That should link to the module API docs as you expect. I personally hate when docs reference API things and don't link to them in the sphinx docs or assume some other structure (it ends up not being accurate as things change). 75% of my contributions to the dask library are just sphinx docs updates like this.

**Review by djhoese — COMMENTED** [review]
Had a few comments. Otherwise, what do you think about a CONTRIBUTING.md document in the root of the repository that is just a sentence linking to the dev install section of the README? That or make a more "official" contributor's guide that the README points to and have the CONTRIBUTING.md that points to that page in the RTD site.

File: docs/conf.py (hunk 2 above)
Comment (almarklein) [reply docs/conf.py:48]: Agreed. I can probably also turn some other references in the guide into links like this.

**Comment by almarklein** [comment]
> Otherwise, what do you think about a CONTRIBUTING.md document in the root of the repository that is just a sentence linking to the dev install section of the README? That or make a more "official" contributor's guide that the README points to and have the CONTRIBUTING.md that points to that page in the RTD site.

Or a CONTRIBUTING.md that describes all things for contributors, and link to that :)

**Pull request merged**